LOAN/RealEstateLoans/HomeMortgageDisclosureActCoveredMortgages.rdf

```diff
@@ -87,66 +87,66 @@
 		<skos:definition>category based on a cultural factors, including nationality, regional culture, ancestry, and language</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-ln-hmda;HMDA-CoveredLoanContract">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-mtg;Mortgage"/>
+	<owl:Class rdf:about="&fibo-loan-rel-hmda;HMDA-CoveredLoanContract">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-rel-mtg;Mortgage"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&lcc-cr;isClassifiedBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-hmda;HowSubmitted"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-rel-hmda;HowSubmitted"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>HMDA covered loan contract</rdfs:label>
 		<skos:definition>a closed-end mortgage loan or open-end line of credit that is not an excluded transaction for HMDA reporting under US section 1003.3(c) of the Revised Home Mortgage Disclosure Act of 2015</skos:definition>
 		<fibo-fnd-utl-av:adaptedFrom>the Revised HMDA regulatory text.</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-ln-hmda;HMDA-Disposition">
+	<owl:Class rdf:about="&fibo-loan-rel-hmda;HMDA-Disposition">
```

Comment:
how is this different from the status of a loan/mortgage application generally?

Reply:
@rivettp - see comments above - we need an SME that is familiar with HMDA regulations, and another that has similar knowledge in the EU, and to address these things in the context of a new FCT.  The goal of this issue was to simply get through the top level ontology, and the updates made to these ontologies were mainly a result of the reorganization of the ontology hierarchy / architecture we did